libr/core/cmd_print.c

```diff
@@ -5356,6 +5356,7 @@ static int cmd_print(void *data, const char *input) {
 	case 'D': // "pD"
 	case 'd': // "pd"
 	{
+		bool segoff = r_config_get_i (core->config, "asm.segoff");
```

Comment:
```suggestion
		bool segoff = r_config_get_b (core->config, "asm.segoff");
```

Reply:
unfortunatelly I need to revert that part because it is more complicated problem than I though at first place. #19599